From: Mattia Rizzolo <mattia@debian.org>
Date: Sat, 21 May 2016 21:55:27 +0000
Subject: skip UnixSocketTest on hurd,
 as unix sockets with SO_REUSEADDR are not supported there

 A little discussion about unix sockets with SO_REUSEADDR can be found on
 https://lists.gnu.org/archive/html/bug-hurd/2016-01/msg00039.html

Forwarded: no
Patch-Name: fix-ftbfs-on-hurd.patch
---
 tornado/test/httpserver_test.py | 2 ++
 1 file changed, 2 insertions(+)

--- python-tornado.orig/tornado/test/httpserver_test.py
+++ python-tornado/tornado/test/httpserver_test.py
@@ -711,6 +711,8 @@
     not hasattr(socket, "AF_UNIX") or sys.platform == "cygwin",
     "unix sockets not supported on this platform",
 )
+@unittest.skipIf(sys.platform == 'gnu0',
+                 "unix sockets with SO_REUSEADDR not supported on this platform")
 class UnixSocketTest(AsyncTestCase):
     """HTTPServers can listen on Unix sockets too.
 
